CodeEdit/Features/CodeFile/CodeFile.swift

```diff
@@ -94,7 +97,7 @@ final class CodeFileDocument: NSDocument, ObservableObject, QLPreviewItem {
     /// This function is used for decoding files.
     /// It should not throw error as unsupported files can still be opened by QLPreviewView.
     override func read(from data: Data, ofType _: String) throws {
-        guard let content = String(data: data, encoding: .utf8) else { return }
+        guard let content = String(data: data, encoding: prefs.preferences.textEditing.defaultTextEncoding) else { return }
```

Comment:
Please make sure to fix the SwiftLint Error on this line.

See [CodeStyle](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style) for reference.

Reply:
i'm aware of this, but thanks for the review!